django_netjsonconfig/utils.py

```diff
@@ -49,6 +49,15 @@ def send_config(config, request):
                      contents=config.generate().getvalue())
 
 
+def send_vpn_config(vpn, request):
+    """
+    returns a ``ControllerResponse``which includes the configuration
+    tar.gz as attachment
+    """
+    return send_file(filename='{0}.tar.gz'.format(vpn.name),
+                     contents=vpn.generate())
```

Comment:
can you add `.getvalue()` for consistency with `send_device_config`?

Eg: `vpn.generate().getvalue()`